recipes/sasl/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+sudo apt-get install -y libsasl2-dev
```

Comment:
We use CentOS for the linux containers so you should add this to the `yum-requirements.txt`. However, I imagine we would prefer that we had our own build of `libsas12`.